src/components/Alert.vue

```diff
@@ -6,6 +6,8 @@
     :type="color"
     :outlined="outlined"
     :prominent="prominent"
+    :text="text"
```

Comment:
Text already used for the message, look at 14 row, for this case i suppouse we need to use another parameter, for example flat, as it was beforein vuetify1

Reply:
Oh sorry I totally miss it, yes, flat sounds good I will edit it.